cirq-ionq/cirq_ionq/__init__.py

```diff
@@ -39,7 +39,7 @@
 
 from cirq_ionq.service import Service as Service
 
-from cirq_ionq.ionq_native_gates import GPIGate as GPIGate, GPI2Gate as GPI2Gate, MSGate as MSGate
+from cirq_ionq.ionq_native_gates import GPIGate as GPIGate, GPI2Gate as GPI2Gate, MSGate as MSGate, ZZGate as ZZGate
 
```

Comment:
Consider if the 2 new native gatesets should be available here as is the IonQTargetGateset.

Reply:
done